src/dev/code_coverage/ingest_coverage/__tests__/enumerate_patterns.test.js

```diff
@@ -35,14 +35,4 @@ describe(`enumeratePatterns`, () => {
       'src/plugins/charts/public/static/color_maps/color_maps.ts kibana-app'
     );
   });
-  it(`should resolve x-pack/plugins/security_solution/public/common/components/exceptions/builder/translations.ts to kibana-security`, () => {
```

Comment:
I'm not too familiar with these tests but could/should this be replaced with an updated path to the `lists` location instead of removing it all together?

Reply:
Hey @dplumlee This test was "unlucky" to randomly select path that you changed. I think you just need to update it with one you have now and we are good. But please don't remove it completely

Reply:
This was actually me. Thanks for taking a look @dmlemeshko . I'll update path and restore test.